gauntlet/packages/gauntlet-serum-multisig/src/commands/setThreshold.ts

```diff
@@ -2,15 +2,13 @@ import { SolanaCommand, RawTransaction, TransactionResponse } from '@chainlink/g
 import { PublicKey } from '@solana/web3.js'
 import { CONTRACT_LIST, getContract } from '@chainlink/gauntlet-solana-contracts'
 import { Result } from '@chainlink/gauntlet-core'
-import { BN } from '@chainlink/gauntlet-core/dist/utils'
+import { BN, logger } from '@chainlink/gauntlet-core/dist/utils'
 
 export default class SetThreshold extends SolanaCommand {
-  static id = 'set:threshold'
+  static id = 'set_threshold'
```

Comment:
```suggestion
  static id = 'multisig:set_threshold'
```

Reply:
It should probably be `multisig:change_threshold` as this is the instruction we call.